dev/refinement-bot/src/sendMessage.ts

```diff
@@ -0,0 +1,201 @@
+import { getOctokit } from '@actions/github'
+import { Repository } from '@octokit/graphql-schema'
+import { KnownBlock, PlainTextOption, SectionBlock } from '@slack/types'
+import dotenv from 'dotenv'
+import fetch from 'node-fetch'
+
+import { GithubIssue, PriorityItem } from './types'
+dotenv.config()
+interface slackMessage {
+    blocks: KnownBlock[] | null
+}
+
+const octokit = getOctokit(process.env.GITHUB_TOKEN as string)
+const issue: string = process.env.RANDOM_ISSUE || process.env.LABELED_ISSUE || ''
+const slackurl: string = process.env.SLACK_WEBHOOK_URL as string
+const checkPriority = !!(process.env.PRIORITY_LIST as string)
+const priorityList: PriorityItem[] = JSON.parse(process.env.PRIORITY_LIST as string) as PriorityItem[]
+if (!issue) {
+    console.assert(issue, 'issue exists')
+    process.exit()
+}
+
+const isPriorityPresent = (json_issue: GithubIssue): boolean => {
+    let priorityPresent = false
+    priorityPresent = !!json_issue.labels?.some(label =>
+        // match p0, p1, p2
+        priorityList.some((item: PriorityItem) => item.value === label?.name)
+    )
+    return priorityPresent
```

Comment:
```suggestion
    if (!json_issue.labels) {
        return false
    }

    return json_issue.labels.some(label =>
        // match p0, p1, p2
        priorityList.some((item: PriorityItem) => item.value === label.name)
    )
```

Good chance where we can just leverage TS so we don't need `!!` to ensure the variable is not undefined